src/Java.Interop.Localization/Resources.resx

```diff
@@ -292,35 +292,41 @@ The following terms should not be translated: Metadata.xml.</comment>
     <comment>{0}, {1} - .NET types.</comment>
   </data>
   <data name="JavaCallableWrappers_XA4200" xml:space="preserve">
-    <value>Cannot generate Java wrapper for type '{0}', only 'class' types are supported.</value>
+    <value>Cannot generate Java wrapper for type '{0}'. Only 'class' types are supported.</value>
     <comment>{0} - Java type.
-The following terms should not be translated: class.</comment>
+The following terms should not be translated:
+class.</comment>
   </data>
   <data name="JavaCallableWrappers_XA4201" xml:space="preserve">
     <value>Cannot determine JNI name for type '{0}'.</value>
     <comment>{0} - Java type.
 The following terms should not be translated: JNI.</comment>
   </data>
   <data name="JavaCallableWrappers_XA4203" xml:space="preserve">
-    <value>The 'Name' property must be a fully qualified 'package.TypeName' value, and no package was found for '{0}'.</value>
+    <value>The 'Name' property must be a fully qualified type like 'com.example.MyClass' and no package was found for '{0}'.</value>
```

Comment:
I believe it's fine to leave it as it is, but just to be excessively thorough, note that the English words `example` and `My` in `com.example.MyClass` will lose a tiny bit of special meaning in the translated versions of the message.

Although the [Microsoft internal FAQ on fictitious names](https://microsoft.sharepoint.com/sites/infopedia/pages/layouts/kcdoc.aspx?k=g07kc-11-186) doesn't specifically mention `example.com`, I suspect that's because it's an even broader case that is accepted industry-wide (https://tools.ietf.org/html/rfc2606) and so doesn't require a specific mention in the FAQ.